template/provision/ansible.cfg.erb

```diff
@@ -1,5 +1,5 @@
 [defaults]
 inventory = hosts
 forks     = 10
-roles_path = <%= File.join(gem_path, 'ansible', 'roles') %>:/etc/ansible/roles
+roles_path = <%= File.join(gem_path, 'ansible', 'roles') %>
```

Comment:
I think we should maybe also still include /etc/ansible/roles? does that mess with anything if they are in there twice?